### PR TITLE
Fix: Track fields changes for creation

### PR DESCRIPTION
**Pull request by rldhont**
Add a method to reset form controls data before tracking form controls changes for creation feature.

### CODE REVIEW

File: lizmap/modules/lizmap/classes/qgisForm.class.php [hunk 1]
```diff
@@ -381,6 +381,25 @@ public function getForm()
         return $this->form;
     }
 
+    /**
+     * Reset the form controls data to Null.
+     *
+     * @return object the Jelix jForm object
```
Comment (laurentj) [inline lizmap/modules/lizmap/classes/qgisForm.class.php:387]: no, this is not a "jForm" object. `object` is too vague. You should put the real class name, `jFormsBase`

Comment (rldhont) [reply lizmap/modules/lizmap/classes/qgisForm.class.php]: Done

**Comment by rldhont** [comment]
@nboisteault can you test it ?

**Review by nboisteault — APPROVED** [review]
Tested OK!

**Comment by rldhont** [comment]
Thanks @nboisteault

**Pull request merged**